prow/config/config.go

```diff
@@ -1919,6 +1940,29 @@ func ValidateController(c *Controller) error {
 	return nil
 }
 
+func ValidateReportTemplates(c *Controller) error {
+	if len(c.ReportTemplateString) > 0 {
```

Comment:
Can we please do empty string checks via `if stringVar == ""`? It makes reading the code easier